File: netcore/Flights/Flights.Core/Model/Flight.cs
using MongoDB.Bson;
using MongoDB.Bson.Serialization.Attributes;

namespace Flights.Core.Model
{
    public class Flight
    {
        [BsonId]
        [BsonRepresentation(BsonType.ObjectId)]
        public string Id { get; set; }
        public string ScheduleTime { get; set; }
        public string Number { get; set; }
        public string Airline{ get; set; }
        public string Origin { get; set; }
        public string Destination { get; set; }
        public string Status { get; set; }
    }
}

File: Java/src/main/java/flights/core/model/Flight.java
package flights.core.model;

import lombok.AllArgsConstructor;
import lombok.Data;

@Data
@AllArgsConstructor
public class Flight {
  private String id;
  private String scheduleTime;
  private String number;
  private String airline;
  private String origin;
  private String destination;
  private String status;
}
